name.abuchen.portfolio/src/name/abuchen/portfolio/snapshot/trades/Trade.java

```diff
@@ -262,7 +247,14 @@ public Money getGrossProfitLoss()
     {
         if (exitGrossValue == null)
             return null;
-        return exitGrossValue.subtract(entryGrossValue);
+        return exitGrossValue.subtract(entryValueNet);
+    }
+
+    public Money getGrossProfitLossMovingAverage()
+    {
+        if (exitGrossValue == null)
+            return null;
```

Comment:
I think we do not need this check anymore since exitGrossValue is now always non-null